operations/routingrules.md

```diff
@@ -0,0 +1,380 @@
+| `Client` | string | required | Name and version of the client application. |
+| `RoutingRuleIds` | array of string | required, max 1000 items | Unique identifier of the [Routing rule](routingrules.md#routing-rule). |
+| `CompanyIds` | array of string | required, max 1000 items | Unique identifier of the [Company](companies.md#company). |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
+
+### Response
+
+```javascript
+{
+    "RoutingRules": [
+        {
+            "Id": "ff785b22-5422-4d1d-87f4-af2e00b3dfda",
+            "CompanyId": "cd441e1a-6f19-4960-887a-af2a00d5d5f8",
+            "CompanyRelation": "PartnerCompany",
+            "ServiceId": "0907a1b4-ef7a-4aa8-b8a1-af2a00d5ca22",
+            "Applicability": "PrepaidOnly",
+            "RouteType": "AllStayItems"
+        },
+        {
+            "Id": "d98c9611-0006-4691-a835-af2e00b170c4",
+            "CompanyId": "ddc23f8d-131d-44d6-b150-af2a00d5d5f8",
+            "CompanyRelation": "TravelAgency",
+            "ServiceId": "0907a1b4-ef7a-4aa8-b8a1-af2a00d5ca22",
+            "Applicability": "Always",
+            "RouteType": "SelectedStayItems",
+            "SelectedStayItems":
+            {
+                "Nights" : true,
+                "CityTax" : true,
+                "ProductCategoryIds":[
+                    "004fa262-7b08-4853-b544-af2a00d5cf1f",
+                    "8f0dc6ef-1fd4-44e4-b353-af2c00b24caf",
+                    "d4c1b435-0ed1-4d98-9833-af2c00b28518"
+                ]
+            }
+        }
+    ],
+    "Cursor": "d98c9611-0006-4691-a835-af2e00b170c4"
+}
+```
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `RoutingRules` | array of [Routing rules](#routing-rule) | required | Routing rules. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### Routing rule
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the service. |
+| `CompanyId` | string | required | Unique identifier of the [Company](companies.md#company) routing rule is assigned to. |
+| `CompanyRelation` | string [Company relation](routingrules.md#company-relation) | required | Relation of the company to the routing rule. |
+| `Applicability` | string [Applicability](routingrules.md#applicability) | required | Determines if routing rule apply to all future reservations with this company or travel agency attached or only future reservations that are prepaid in online travel agency (OTA). |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) routing rule is assigned to. |
+| `RouteType` | string [Route type](routingrules.md#route-type) | required | What should be routed. |
```

Comment:
Gotta come up with a better wording, this isn't quite obvious. `What should be routed.` says very little and is too abstract.